db/migrate/20200623144214_change_minor_update_boolean_to_update_type_string.rb

```diff
@@ -0,0 +1,11 @@
+class ChangeMinorUpdateBooleanToUpdateTypeString < Mongoid::Migration
+  def self.up
+    TravelAdviceEdition.where(minor_update: true, update_type: nil).update_all(update_type: "minor")
+    TravelAdviceEdition.where(minor_update: false, update_type: nil).update_all(update_type: "major")
+  end
+
+  def self.down
+    TravelAdviceEdition.where(update_type: "minor").update_all(minor_update: true)
```

Comment:
```suggestion
    TravelAdviceEdition.where(update_type: "minor").update_all(minor_update: true, update_type: nil)
```

Pretty minor, but we wouldn't be able to run the forward migration correctly again if we had to rollback, since we would no longer have `update_type: nil` as a flag.